spec/abci/abci++_methods.md

```diff
@@ -749,11 +749,13 @@ Most of the data structures used in ABCI are shared [common data structures](../
 * **Fields**:
 
     | Name      | Type                                                          | Description                                                                  | Field Number | Deterministic |
-    |-----------|---------------------------------------------------------------|------------------------------------------------------------------------------|--------------|---------------|
+    | --------- | ------------------------------------------------------------- | ---------------------------------------------------------------------------- | ------------ | ------------- |
     | block     | [BlockParams](../core/data_structures.md#blockparams)         | Parameters limiting the size of a block and time between consecutive blocks. | 1            | Yes           |
     | evidence  | [EvidenceParams](../core/data_structures.md#evidenceparams)   | Parameters limiting the validity of evidence of byzantine behaviour.         | 2            | Yes           |
     | validator | [ValidatorParams](../core/data_structures.md#validatorparams) | Parameters limiting the types of public keys validators can use.             | 3            | Yes           |
     | version   | [VersionsParams](../core/data_structures.md#versionparams)    | The ABCI application version.                                                | 4            | Yes           |
+    | abci      | [ABCIParams](../core/data_structures.md#abciparams)           | ABCI-related parameters.                                                     | 5            | Yes           |
```

Comment:
Mentioning for tracking but I think this was discussed in our session that this doesn't belong in the pr

Reply:
Thanks @glnro, that is true. Let me revert it and add it in a different PR